src/components/views/CreateViewActionButton.tsx

```diff
@@ -45,7 +45,7 @@ const CreateViewActionButton: React.FunctionComponent = () => {
                     data-testid="create-view-action-button"
                     onClick={ () => {
                         NProgress.start();
-                        createNewViewMutation.mutate();
+                        createNewViewMutation.mutate(undefined);
```

Comment:
I feel like this shouldn't need to be explicit if it's an optional argument?

Reply:
I couldn't get that to work, same issue we were having with the separate API mutation factories for delete vs patch/post/put. But I just realized how to solve it now!

Reply:
Hint: The solution was to allow `void` as an optional (union) type, see bc619cb4bb3e78a9c6ce5497d66ccd1c60befd90.